src/Bridge/Doctrine/EventListener/ManagerViewListener.php

```diff
@@ -65,8 +66,11 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
                 $event->setControllerResult(null);
             break;
         }
-
-        $objectManager->flush();
+        try {
+            $objectManager->flush();
+        } catch (\Exception $e) {
```

Comment:
How can I catch a more precise exception ? 

Like @teohhanhui said, we need to catch `Doctrine\ORM\EntityNotFoundException`.